index.js

```diff
@@ -167,7 +167,7 @@ function createOrginisationViewHtml() {
  * @description Creates the HTML for the student view and sets it as the innerHTML of the section2Container element.
  */
 function createStudentViewHtml() {
-  if (studentIsLoggedIn) {
+  if (!studentIsLoggedIn) {
```

Comment:
Question. is this line supposed to show the student view when user is not logged in/authenticated?

Reply:
@Christonn93

Reply:
Did some changes now to get what i wanted., 
![image](https://user-images.githubusercontent.com/46559129/214775177-87c8c5ed-caf9-412f-a47a-211102b79511.png)

I need to ask the creator of what he was planning on when he made this code. but now it will show the correct layout
![image](https://user-images.githubusercontent.com/46559129/214775296-c6babf67-6e76-4b1b-bd80-cd123ddb2d09.png)